### ISSUE TITLE
CF test throws error for "orthogonal" DSG TimeSeries

### ISSUE THREAD

**Issue by pacioos** [ioos/compliance-checker#469]
When running the CF test of cchecker.py on an "orthogonal" CF DSG TimeSeries dataset, the checker throws the following error:

> time shares the same name as one of its dimensions

In the "orthogonal" representation documented in both the [CF spec](http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html#_orthogonal_multidimensional_array_representation_of_time_series) and [NODC/NCEI "gold standard"](https://www.nodc.noaa.gov/data/formats/netcdf/v2.0/timeSeriesOrthogonal.cdl) templates, the variable "time" is a coordinate variable with dimension "time". This should be a valid form of a DSG TimeSeries, so I believe the above error should not occur? It would seem the checker is enforcing the "incomplete" (non-orthogonal multidimensional array) format of CF DSG TimeSeries in which "time" is the variable and "obs" (or other) is the dimension name, as documented in these [CF](http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html#_incomplete_multidimensional_array_representation_of_time_series) and [NCEI](https://www.nodc.noaa.gov/data/formats/netcdf/v2.0/timeSeriesIncomplete.cdl) templates; in this case, I am not given this error. I believe the checker needs to allow both forms as equally valid.
Thanks,
John Maurer

**Comment by lukecampbell** [comment]
Do you have an example?

**Comment by lukecampbell** [comment]
I just ran the compliance checker web on the time series gold standard and I don't see this error.

[GoldStandard.pdf](https://github.com/ioos/compliance-checker/files/743022/GoldStandard.pdf)


Is this the right one?

**Comment by pacioos** [comment]
Yes, that is the right example, @lukecampbell. Sorry, I overlooked the fact that "time" is *not* multidimensional in this case, though the physical variables are (e.g., "temp", "sal", etc.). While the "time" variable is a coordinate variable with dimension "time" as I described above, it does not also have the "timeseries" dimension in the "orthogonal" format. The fact that we had "time(timeseries, time)" rather than "time(time)" or "time(timeseries, obs)" is the root of our problem. This issue can be closed as it is user error, not a cchecker.py oversight. Thanks for clearing this up for me!
